src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started her work.");
```

Comment:
The message should be more informative and indicate the type of machine that is working. For example, 'Excavator started its work.'